lib/colorado_lottery.rb

```diff
@@ -0,0 +1,52 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants,
+              :winners,
+              :current_contestants
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants= Hash.new { |hash, key| hash[key] = [] }
+  end
+
+  def interested_and_18?(contestant, game)
+    interested = contestant.game_interests.any? { |interested| interested == game.name }
```

Comment:
this works well! the one small point is that `any` is any enumerable, and we try to avoid enumerating unnecessarily when possible (it's no big deal with small collections, but when we have large data sets, it matters a lot).  `include?` is a preferable tool here for that reason.  I do like that you split it into the interested local variable for readability/line length. 


```
def interested_and_18?(contestant, game)
      contestant.age >= 18 && contestant.game_interests.include?(game.name)
  end
```